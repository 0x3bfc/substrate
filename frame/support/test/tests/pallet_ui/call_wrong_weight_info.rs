#[frame_support::pallet]
mod pallet {
	use frame_support::pallet_prelude::Hooks;
	use frame_system::pallet_prelude::{BlockNumberFor, OriginFor};

	#[pallet::config]
	pub trait Config: frame_system::Config {}

	#[pallet::pallet]
	pub struct Pallet<T>(core::marker::PhantomData<T>);

	#[pallet::hooks]
	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {}

	#[pallet::call]
	#[pallet::weight_info(not a type)]
	impl<T: Config> Pallet<T> {
	}
}

fn main() {
}
