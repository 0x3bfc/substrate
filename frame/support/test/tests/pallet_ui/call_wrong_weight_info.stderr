error: unexpected token
  --> $DIR/call_wrong_weight_info.rs:16:28
   |
16 |     #[pallet::weight_info(not a type)]
   |                               ^
